### PR TITLE
✨ : Add commit hash and release date indicator

### PR DESCRIPTION
**Pull request by sasamuku**
## Summary

Add commit hash and release data  for ERD Web.

`cliVersion` will be refactored to more reasonable in next PR.

#### ERD Web

sample url for checking:
https://liam-erd-c635e64zt-route-06-core.vercel.app/erd/p/raw.githubusercontent.com/companionstudio/islay-integration/refs/heads/master/db/structure.sql

image

#### CLI (No Changes)

image

### CODE REVIEW

File: frontend/packages/erd-core/src/components/ERDRenderer/AppBar/HelpButton/ReleaseVersion.tsx [hunk 1]
```diff
@@ -16,12 +16,22 @@ export const ReleaseVersion: FC = () => {
...
+            {' '}
+            {cliVersion.isReleasedGitHash ||
+              `+ ${cliVersion.gitHash.slice(0, 7)} `}
+          </span>
+          <span>{cliVersion.date && ` (${cliVersion.date})`}</span>
+        </>
+      ) : (
+        <>
+          <span>{cliVersion.gitHash.slice(0, 7)}</span>
+          <span>{cliVersion.date && ` (${cliVersion.date})`}</span>
```
Comment (sasamuku) [inline frontend/packages/erd-core/src/components/ERDRenderer/AppBar/HelpButton/ReleaseVersion.tsx:32]: release date is actually unnecessary but the UI looks better if it is included.

no release date:

image

File: frontend/packages/erd-core/src/schemas/cliVersion/schemas.ts [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
   envName: v.string(),
   isReleasedGitHash: v.boolean(),
   date: v.string(),
+  displayedOn: v.literal('cli'),
 })
+
+export const cliVersionSchema = v.union([
+  webVersionSchema,
+  innerCliVersionSchema,
+])
```
Comment (sasamuku) [inline frontend/packages/erd-core/src/schemas/cliVersion/schemas.ts:21]: used discriminated union👍
e897cc63d58e57b7eb5d2fcddae8f3667250f43a

Comment (sasamuku) [reply frontend/packages/erd-core/src/schemas/cliVersion/schemas.ts:21]: @MH4GF 
I've refactored cliVersion type definition, pls re-review👍

Comment (sasamuku) [reply frontend/packages/erd-core/src/schemas/cliVersion/schemas.ts:21]: Oh, I forgot I should refactor 😅
How about next PR?

image

Comment (MH4GF) [reply frontend/packages/erd-core/src/schemas/cliVersion/schemas.ts:21]: @sasamuku Oh! OK, next PR will be fine!

**Review by MH4GF — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**